### PR TITLE
Add pre commit cli code

### PR DESCRIPTION
**Pull request by esequielvirtuoso**
# What?

Add pre commit cli code.

# Why

That will prevent developers from having to rebase your commit history to adapt the commit messages to [semantic-release](https://github.com/NeowayLabs/semantic-release) standards.

### CODE REVIEW

File: pre_commit.go [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+func printRed(text string) {
+	fmt.Println(colorRed + text + colorReset)
+}
+
+func printGreen(text string) {
+	fmt.Println(colorRed + text + colorReset)
+}
+
+func getCommitType() (string, error) {
+	printGreenYellow("[COMMIT TYPE]", ` Which one is your commit type? *insert the option number*
```
Comment (ebbauer) [inline pre_commit.go]: @esequielvirtuoso I think you should use white instead. it looks like a warning message

File: pre_commit.go [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+		1.              [fix]: A code/bug fix;
+		2.             [docs]: Documentation only changes;
```
Comment (ebbauer) [inline pre_commit.go]: @esequielvirtuoso documentation only changes? what do you mean? maybe `Only documentation changes`?

**Review by ebbauer — APPROVED** [review]
nice! just some minor adjustments

File: pre_commit.go (hunk 2 above)
Comment (esequielvirtuoso) [reply pre_commit.go]: Yeah, I'll change it.

**Review by julianogalgaro — APPROVED** [review]
LGTM!

**Pull request merged**